src/azure-api-utils.ts

```diff
@@ -25,7 +25,7 @@ function bitAfter(bits: string[], after: string): string | undefined {
     return bits[afterIndex + 1];
 }
 
-export function toSubscription(context: ISubscriptionContext): SubscriptionModels.Subscription {
+export function toSubscription(context: ISubscriptionContext): Subscription {
```

Comment:
I see that some uses of this are now removed. Is it still used anywhere? Wondering if it can be removed...